pkgs/applications/misc/lutris/default.nix

```diff
@@ -0,0 +1,179 @@
+    libxml2 libusb libpng libmpeg2 libv4l libjpeg libxkbcommon libass libcdio
+    libjack2 libsamplerate libzip libmad libaio libcap libtiff libva libgphoto2
+    libxslt libtxc_dxtn libsndfile giflib zlib glib alsaLib zziplib bash dbus
+    keyutils zip cabextract freetype unzip coreutils readline gcc SDL SDL2 curl
+    graphite2 gtk2 gtk3 udev ncurses wayland
+
+    # Lutris
+    gobjectIntrospection gdk_pixbuf pango openssl sqlite xterm
+
+    # Adventure Game Studio
+    allegro dumb
+
+    # Desmume
+    lua agg soundtouch openal desktop-file-utils pangox_compat atk
+
+    # DGen // TODO: libarchive is broken
+
+    # Dolphin
+    bluez ffmpeg gettext portaudio wxGTK30 miniupnpc mbedtls lzo sfml gsm
+    wavpack gnutls-kdh orc nettle gmp pcre vulkan-loader
+
+    # DOSBox
+    SDL_net SDL_sound
+
+    # Higan // TODO: "higan is not available for the x86_64 architecture"
+
+    # Libretro
+    fluidsynth hidapi mesa_noglu libdrm
+
+    # MAME
+    qt48 fontconfig SDL2_ttf
+
+    # Mednafen
+    freeglut mesa_glu
+
+    # MESS
+    expat
+
+    # Mupen64Plus
+    boost dash
+
+    # Osmose
+    qt4
+
+    # PCSX2 // TODO: "libgobject-2.0.so.0: wrong ELF class: ELFCLASS64"
+
+    # PPSSPP
+    glew snappy
+
+    # Redream // "redream is not available for the x86_64 architecture"
+
+    # ResidualVM
+    flac
+
+    # rpcs3 // TODO: "error while loading shared libraries: libz.so.1..."
+    llvm_4
+
+    # ScummVM
+    nasm sndio
+
+    # Snes9x
+    epoxy minizip
+
+    # Steam
+    steam
+
+    # Vice
+    bison flex
+
+    # WINE
+    perl which p7zip gnused gnugrep psmisc cups lcms2 mpg123 cairo unixODBC
+    samba4 sane-backends openldap opencl-headers ocl-icd utillinux
+
+    (wine.override { wineBuild = "wineWow"; })
+
+    # ZDOOM
+    soundfont-fluid bzip2 game-music-emu
+  ] ++ qt5Deps
+    ++ gnome2Deps
+    ++ gnome3Deps
+    ++ python2Deps
+    ++ python3Deps
+    ++ xorgDeps
+    ++ gstDeps;
+
+  lutris = python3Packages.buildPythonApplication rec {
+    name = "lutris-${version}";
+    version = "v0.4.18";
+
+    src = fetchFromGitHub {
+      owner = "lutris";
+      repo = "lutris";
+      rev = version;
+      sha256 = "1pgvk3qaaph1dlkrc5cq2jifr3yqlhnqsfa0wkaqzssh9acd5q9b";
+    };
+
+    patches = [(fetchpatch {
+      url = "https://github.com/lutris/lutris/commit/403a83e18690511cb723f883254c86081870f050.patch";
+      sha256 = "1zc45jgaqvpqgr6f9mkqzcwx48lffrjkp7zfk0fz4l2a05jim6hm";
+    })];
+
+    enableParallelBuilding = true;
+    nativeBuildInputs = [ wrapGAppsHook ];
+    propagatedBuildInputs = deps;
+    python2Path = python2Deps;
+    fullPath = stdenv.lib.makeLibraryPath deps;
+    preConfigure = "export HOME=$PWD";
+
+    makeWrapperArgs = [
+      "--prefix LD_LIBRARY_PATH : ${fullPath}:$out/lib"
+      "--set GI_TYPELIB_PATH $GI_TYPELIB_PATH"
+      "--prefix XDG_DATA_DIRS : $out/share"
+      "--suffix XDG_DATA_DIRS : $XDG_ICON_DIRS:$GSETTINGS_SCHEMAS_PATH"
```

Comment:
GSETTINGS_SCHEMAS_PATH will be also set automatically by `wrapGAppsHook` when `glib gnome3.gsettings_desktop_schemas` are in `buildInputs`: https://github.com/NixOS/nixpkgs/blob/master/pkgs/applications/networking/instant-messengers/rambox/default.nix#L33